be/src/runtime/load_channel_mgr.cpp

```diff
@@ -213,11 +218,16 @@ Status LoadChannelMgr::_start_bg_worker() {
 
 Status LoadChannelMgr::_start_load_channels_clean() {
     std::vector<std::shared_ptr<LoadChannel>> need_delete_channels;
+    const int32_t max_alive_time = config::streaming_load_rpc_max_alive_time_sec;
+    LOG(INFO) << "start cleaning load channel which is not active for more than " << max_alive_time << " seconds";
     time_t now = time(nullptr);
     {
         std::vector<UniqueId> need_delete_channel_ids;
         std::lock_guard<std::mutex> l(_lock);
+        LOG(INFO) << "there are " << _load_channels.size() << " running load channels";
+        int i = 0;
         for (auto& kv : _load_channels) {
+            LOG(INFO) << "load channel[" << i++ << "]: " << *(kv.second);
```

Comment:
I think there may be too many logs which is not be worthful

Reply:
Periodically dump information about running load channels may be useful in some cases. I'll change to use VLOG for it.